### ISSUE TITLE
loss D is zero from the beginning

### ISSUE THREAD

**Issue by Nick-Sun1018** [LynnHo/CycleGAN-Tensorflow-Simple#37]
Thanks for your excellent implementation, I want to translate synthetic haze to real haze, and when I training cyclegan with my dataset, the loss of both D is zero from the beginning, and the quality of the translated image is not as good as I expected, how to improve this?
here's the loss of D from tensor board
![屏幕快照 2021-06-23 上午10 30 09](https://user-images.githubusercontent.com/61535862/123026269-74d9e600-d40e-11eb-8d8e-d715132b1ae2.png)
and here's my dataset
this is syn_haze
![haze_1](https://user-images.githubusercontent.com/61535862/123026501-e2861200-d40e-11eb-975d-0e3b79c4be6f.jpg)
and this is real_haze
![JL_Google_033](https://user-images.githubusercontent.com/61535862/123026560-f92c6900-d40e-11eb-9194-591685da5c74.jpeg)
